electroncash_plugins/fusion/qt.py

```diff
@@ -317,7 +317,9 @@ def utxo_list_item_setup(self, utxo_list, item, utxo, name):
             item.setText(col, _("Partial {count}/{total}").format(count=count, total=fuse_depth))
             item.setIcon(col, icon_fusion_logo_gray)
         elif self.is_fuz_address(wallet, utxo['address'], require_depth=fuse_depth-1):
-            item.setText(col, _("Fusion Addr"))
+            item.setText(col, _("Fusion Address"))
```

Comment:
Hmm.. it kind of sticks out on the right too far.

<img width="300" alt="Screen Shot 2021-08-16 at 8 51 50 PM" src="https://user-images.githubusercontent.com/266627/129607559-71345c0b-8162-472a-8620-37ea6ffa6020.png">


Mind just making the text shorter?

```suggestion
            item.setText(col, _("Fuse Addr"))
```